lib/rspec-console/exceptions.rb

```diff
@@ -0,0 +1,8 @@
+module RSpecConsole
+  class VersionError < StandardError
+    'Please use RSpec 2.9.10 or later'
```

Comment:
This is in expression in a class, it has no effect on the code whatsoever. You should put this in a `to_message` or something. Or make the caller call something like `raise VersionError, "Please use RSpec 2.9.10 or later"`